File: Back/src/ProEventos.Domain/Speecher.cs
using System.Collections.Generic;

namespace ProEventos.Domain
{
    public class Speecher
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Resume { get; set; }
        public string ImageURL { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<SpeecherEvent> SpeecherEvents { get; set; }



    }
}

File: Front/ProEventos-App/src/app/models/Speecher.ts
import { Event } from './Event';
import { SocialNetwork } from './SocialNetwork';

export interface Speecher {
  id: number;
  name: string;
  resume: string;
  imageUrl: string;
  phoneNumber: string;
  email: string;
  socialNetworks: SocialNetwork[];
  speecherEvents: Event[];
}
